persons: fix filter to get documents in organisation views

benerken opened the pull request:
## Why are you opening this PR?

- Uses correct filter (holdings in place of items)
  to display documents list in person detailed views
  filtered by organisation views.
- Closes #553

Co-Authored-by: Benoit Erken erken.benoit@gmail.com

## How to test?

- In a person detailed view, I must see a link to a document detailed view, both in the global view and in the organisation view

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

iGormilhit reviewed (changes requested):
Commit message: reference to the taiga task is not needed. And try to be more specific:

```
persons: fix filter to get documents in person view

* Uses correct filter (holdings in place of items)
  to display documents list in person detailed views
  filtered by organisation views.
* Closes #553
```
Or something like that. Thanks.

benerken committed:
persons: fix filter to get documents in organisation views

* Uses correct filter (holdings in place of items)
  to display documents list in person detailed views
  filtered by organisation views.
* Closes #553

Co-Authored-by: Benoit Erken erken.benoit@gmail.com
